### ISSUE TITLE
Cannot find scaler.p

### ISSUE THREAD

**Issue by prodigystd** [mvirgo/MLND-Capstone#2]
I tried to run this lane_lines.py and got the following error:

Traceback (most recent call last):
  File "lane_lines.py", line 26, in <module>
    label_scaler = pickle.load(open( "scaler.p", "rb" ))
FileNotFoundError: [Errno 2] No such file or directory: 'scaler.p'

**Comment by mvirgo** [comment]
Yes, this was part of the old method used to normalize the lane line coefficients - this is not part of the final solution I used. I'd suggest commenting out this line and line 58 (where the scaler is called) to use the file as is without needing the related scaler.

**Comment by prodigystd** [comment]
Thanks for prompt response, I have already tried it. But I wasn't satisfied by the performance of the lane detection using this file. So, I found and run the draw_detected_lines.py and it works pretty well with my raw video. Only now I understood the first file lane_lines.py was different approach and draw_detected_lines.py implements an improved method which applies cnn directly to the image without transform.
